website/docs/registry/faq.mdx

```diff
@@ -0,0 +1,124 @@
+
+There may be situations where it is important that a version be removed from the Registry. Perhaps a version of a provider has a critical security vulnerability, or you may have extremely high confidence that no one is using a version of your module that doesn't work as you intended it to.
+
+In these situations, your options for removing previously published versions vary depending on the type of integration:
+
+- **Modules**: The owner of a module can use the Terraform Registry website to remove a specific version, all versions of a specific module provider, or remove the module entirely. [More details on this workflow can be found here](/terraform/registry/modules/publish#removing-published-content).
+- **Providers**: There is no manual process for removing a version of a provider. [Reach out to us via email](/terraform/registry#getting-help) if it is critical that a version be deleted.
+- **Policy Libraries**: The owner of a policy library can completely remove it from the Registry, through the website. [More details on this workflow can be found here](/terraform/registry/policy-libraries/publishing#removing-a-policy-library).
+
+### How can I correct a bug, typo, or documentation error in a version of my provider, module, or policy library once it has been published?
+
+If there is a problem with a certain version, the best resolution is to publish a new release with the issue fixed, incrementing the patch version according to [standard Semantic Versioning practices](https://semver.org/#spec-item-6) (e.g. from `1.2.0` to `1.2.1`).
+
+### Can I remove a version of a provider, module, or policy library by deleting the release or tag from GitHub?
+
+**We recommend you never remove content from a GitHub repository that has already been published to the Terraform Registry**.
+
+The Terraform Registry acts as a kind of index for Terraform-related content that is hosted on GitHub. When you make a new tag or a new release in a GitHub repository that holds your Terraform integration, the Registry automatically creates an index record for a new content. Think of that index record as simply a pointer to the files that exist in your GitHub repository.
+
+If you remove a file, or a tag, a release, or other content from GitHub, the Registry will not be made aware that you did so. The Registry may still host index records pointing to the now-removed files or content. As a result, someone using your provider or module may see their Terraform configuration break, since through the Registry it expects to find files on GitHub which are no longer available.
```

Comment:
Rewriting in active voice and removed a few words for brevity:
```suggestion
The Terraform Registry is unaware if you remove a file, a tag, a release, or other content from GitHub. The Registry could still host index records pointing to now-removed content. As a result, someone using your provider or module may have their Terraform configuration break because the Registry is referencing GitHub files that are no longer available.
```